app/src/main/res/drawable/ic_star.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@saketkumar95 Here are the reasons of choosing vector drawable here: 
1. Vector drawables reduce the size of apk. Though it doesn't matter if the number is less but it is a mattern of concern if number starts increasing.
2. Vector drawable scales better than any other format irrespective of the screen resolution.
3. Apart from all this, we can play with vector drawables in real time and change their properties(like color, etc.).

I hope you understand :)